pkg/packagekit/manifest.json.in

```diff
@@ -8,7 +8,7 @@
 
     "tools": {
         "index": {
-            "label": "Software Updates",
+            "label": "Software updates",
```

Comment:
If we adopt the navigation as title case PF4 recommendation, then this should remain Title Case.